codeHF/dpl-config_run5_hf.json

```diff
@@ -518,12 +598,46 @@
         "d_selectionFlagLc": "0"
     },
     "hf-task-xic": {
-        "cutEtaCandMax": "0.8",
-        "d_selectionFlagXic": "0"
+        "cutEtaCandMax": "-1",
```

Comment:
This value needs to be updated, according to what used of the Lc analysis.

Reply:
I think It will be great to update the Xicc cuts after the checking of sig and bkg distributions which obtained by realistic Xicc pT spectrum.

Reply:
@ginnocen What is the conclusion on this one?